src/managers/KeybindManager.cpp

```diff
@@ -2166,6 +2172,19 @@ void CKeybindManager::moveIntoGroup(std::string args) {
     PWINDOW->m_dWindowDecorations.emplace_back(std::make_unique<CHyprGroupBarDecoration>(PWINDOW));
 }
 
+void CKeybindManager::global(std::string args) {
```

Comment:
why is this moved

Reply:
It's been a long day

Reply:
long like my dicc hehehehe (just kidding I have a very small penis)